addons/test_mail_sms/tests/test_phone_blacklist.py

```diff
@@ -51,6 +61,26 @@ def test_phone_blacklist_internals(self):
             rec = self.env['mail.test.sms.bl'].search([('phone_sanitized_blacklisted', '=', True)])
             self.assertEqual(rec, self.env['mail.test.sms.bl'])
 
+    def test_phone_blacklist_unblacklisted(self):
```

Comment:
Shouldn't we add a small test for ``_add`` and check it effectively adds a log ? Looking at the file it does not seem to be tested, could be the opportunity to cover it ...

Reply:
i couldn't find use case to base such test case on, because:
- when new blacklist is added we use create method 
- when existing (unblacklisted) record is blacklisted we use add method, but no message is used in that flow

So that leads me to the following:
**I may have miss something, but it seems to me that this logging won't be used at all**

Reply:
It is used at least in email marketing normally. A message is posted telling it was a manual blacklist, ...

Reply:
can you elaborate? phone validation used in email marketing?

Reply:
Blacklist mechanism is used in email marketing: people can blacklist themselves to stop receiving news.

Reply:
Wait, talking about mail.blacklist, not phone.blacklist :facepalm:  forgot we did not update SMS marketing.

Reply:
Found something: phone.blacklist.remove wizard adds a message when removing which is basically the same rationale as _add .

Reply:
yes, that would makes sense to add test checking if message was logged with `PhoneBlackList._remove`
tho I'm not how to structure that test, I'd need to use some MessageGate (mock) to capture that log right?

Reply:
Thhnking of it, better close the bug, otherwise we will never finish it :D